src/components/common/infocolourBar/InfoColourbar.tsx

```diff
@@ -2,20 +2,21 @@ import React from 'react'
 import './infoColourBar.scss'
 
 interface InfoColourBarType {
-    message: string
+    message: React.ReactNode
```

Comment:
It won't impact the existing implementation.

Reply:
Yup. It'll not impact the existing implementation as ReactNode has ReactChild as one of the types which at the end has ReactText (string | number) as one of the types 🙂